.github/workflows/deno-ci.yml

```diff
@@ -38,3 +38,11 @@ jobs:
         with:
           file: ./lcov.info
           token: ${{ secrets.CODECOV_TOKEN }}
+                        
+      - name: Report health score
+        uses: slackapi/slack-health-score@v0.0.2
+        with:
+          codecov_token: ${{ secrets.FILS_CODECOV_API_TOKEN }}
```

Comment:
Is this the correct token to use?

Reply:
Yeah I just wanted to make it clear which human user the API token belongs to (in case I e.g. get hit by a bus)